packages/drivers/odsp-driver/src/getFileLink.ts

```diff
@@ -111,11 +115,24 @@ async function getFileLinkCore(
                 };
                 const response = await fetchHelper(url, requestInit);
                 additionalProps = response.commonSpoHeaders;
-                if (response.content.ok) {
-                    const sharingInfo = await response.content.json();
-                    return sharingInfo?.d?.directUrl as string;
+
+                // We actually already know the response is ok, or else the fetchHelper would have thrown
+                // (it does an even more thorough check).  Keeping it here though for layers of protection.
+                if (!response.content.ok) {
```

Comment:
All the places where you check response.content.ok - I thought fetchHelper aldready does it, and this code thus is not reachable. Can you please double check?

Reply:
That's correct, and I am OK removing these if you prefer.  These checks were already in place so I kept them around, but they shouldn't be necessary.

Reply:
I'd remove them - I do not like dead code :)

Reply:
Removed in latest.